webpack/app/widgets/queryselect/widget.js

```diff
@@ -236,7 +235,13 @@ class QuerySelectWidgetController extends React.Component {
    * @returns {Boolean} true/false if the search field is rendered
    */
   show_search_field() {
-    if (!this.state.multi_valued && this.state.values.length > 0 && this.state.hide_input_after_select) {
+    if (this.state.disabled) {
+      return false;
+    }
+    if (this.state.readonly) {
+      return false;
+    }
+    if (!this.state.multi_valued && this.state.values.length > 0) {
       return false;
```

Comment:
Not sure about this. Probably the last line needs to be first to keep the input widget visible, even if it is readonly/disabled.
Or maybe just return `True` if `this.state.multi_valued`?

Reply:
Why one would want the input element to be displayed if it cannot be used (because is disabled or readonly)?

Reply:
Hmm, good point. To see that it is disabled by purpose maybe?